config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
   root to: 'application#index'
 
+  require 'sidekiq'
   require 'sidekiq/web'
+  
+  Sidekiq::Web.use Rack::Auth::Basic do |username, password|
+    ActiveSupport::SecurityUtils.secure_compare(::Digest::SHA256.hexdigest(username), ::Digest::SHA256.hexdigest(ENV.fetch('nombre_de_la_key', rails-bootstrap))) &
```

Comment:
`'nombre_de_la_key'`...rename this please.

Reply:
@francoprud :cat: